extra/integration-testing/test-compat/docker-compose.compat-mender-3.4.yml

```diff
@@ -0,0 +1,12 @@
+version: '2.1'
+services:
+  mender-client-3-4:
+    image: mendersoftware/mender-client-qemu:mender-3.4
+    networks:
+       - mender
+    stdin_open: true
+    tty: true
+    privileged: true
+    environment:
+      TENANT_TOKEN: ""
+
```

Comment:
Technically speaking this one is not necessary here, as this is the version that is tested in every other test. But it is harmless so :ok_hand: